site/assets/js/snippets.js

```diff
@@ -96,6 +96,15 @@
     })
   }
 
+  // --------
+  // Carousels
+  // --------
+  // Instantiate all non-autoplaying carousels in a docs or StackBlitz page
+  document.querySelectorAll('.carousel:not([data-bs-ride="carousel"])')
+    .forEach(carousel => {
+      bootstrap.getOrCreateInstance.Carousel(carousel)
```

Comment:
When loading https://deploy-preview-37354--twbs-bootstrap.netlify.app/docs/5.2/components/carousel/, we have this in the console:

```
Uncaught TypeError: bootstrap.getOrCreateInstance is undefined
    <anonymous> https://deploy-preview-37354--twbs-bootstrap.netlify.app/docs/5.2/assets/js/docs.min.js:6
    <anonymous> https://deploy-preview-37354--twbs-bootstrap.netlify.app/docs/5.2/assets/js/docs.min.js:6
    <anonymous> https://deploy-preview-37354--twbs-bootstrap.netlify.app/docs/5.2/assets/js/docs.min.js:6
```

Isn't it `bootstrap.Carousel.getOrCreateInstance(carousel)` here?

Reply:
Tried a fix via https://github.com/twbs/bootstrap/pull/37354/commits/7fd7a3b8b672f7d351115c7aecc43628d34d4fbd. Please revert it if it causes any issue 🙏